components/AddressContainer.tsx

```diff
@@ -48,19 +48,12 @@ export const AddressContainer: FC<{
     }, 2 * 1000);
   };
 
-  const icon = showCheck ? (
-    <FiCheckSquare className={styles.icon} />
-  ) : (
-    <FiCopy className={styles.icon} />
-  );
+  const icon = showCheck ? <FiCheckSquare className={styles.icon} /> : <FiCopy className={styles.icon} />;
```

Comment:
I kind of like how you had it spaced before, this feels a bit weird now with the closing tage for FiCheckSquare trailing